packages/block-editor/src/components/plain-text/index.js

```diff
@@ -10,15 +10,41 @@ import { forwardRef } from '@wordpress/element';
 import TextareaAutosize from 'react-autosize-textarea';
 import classnames from 'classnames';
 
+/**
+ * Constants
+ */
+const REGEXP_NEWLINES = /[\r\n]+/g;
+
 /**
  * @see https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/plain-text/README.md
  */
-const PlainText = forwardRef( ( { onChange, className, ...props }, ref ) => {
+const PlainText = forwardRef( ( {
+	onChange,
+	className,
+	multiline,
+	isStylable,
```

Comment:
Should this be a prop, or should this be by default? I mean if users of `PlainText` don't want the styled behavior, they can always use TextAreaAutosize or just set styles on the wrapper? or do you think this behavior is too common to warrant a prop?

Reply:
This is a good question.
Mainly, I wanted to affect as little as possible the other components using `PlainText`, to avoid introducing visual regressions that might be hard to catch.

At the same time, those other components currently use `PlainText` exactly like a normal `textarea` (plus the auto-resize, of course)... 🤔